string name
string value
string no_notify
---
bool badValue
